data/coalition/resistance intro.txt

```diff
@@ -1661,27 +1806,37 @@ mission "Resistance: Quarg Interview"
 					goto outsiders
 				`	"How do you pay for all this?"`
 					goto pay
-				`	"I'm fine for now. What's about this task you need me for?"`
+				`	"I'm fine for now. What about this task you need me for?"`
 					goto task
+			action
+				log "Factions" "Lunarium Redoubt" "The Lunarium expect the Heliarchs to fight fiercely, and that in removing them, much damage will be doen to the Coalition. So, their first intention after freeing it from their rule is to rebuild. They also wish to ensure whatever system of governance comes after them is more transparent."
 			`	"Rebuild, first we will need to," Chiree responds. "Fight tooth and nail to remain in power, they will, so expect a great deal of damage done to the Coalition, we do."`
 			`	"Once back in order, everything is, focus on reassuring the populace we should. A change for the better, it will be, and help in making sure everyone realizes that, House Bebliss will," Lobmab jumps in. "Earn back what they took from us, we will."`
 			`	Once the head of House Bebliss is done, Chiree continues. "Pointless, our fight will have been, if corrupted in those rings again, our successors thousands of years from now become. Set up transparency measures, and means to ensure that absolute power, the consuls do not have, we will."`
 				goto questions
 			label fight
+			action
+				log "Factions" "Lunarium Redoubt" "The Lunarium is, military, at a great disadvantage to the Heliarchs. Their ships are far less durable and their weapons far less powerful. Elirom, however, believes that the Lunarium are smarter than the Heliarchs, and that they are only a few steps away from having the ability to close the technological gaps as well, though Pyakri is skeptical of his words."
```

Comment:
Other than the military thing, it's not the case that Elirom thinks the Lunarium is smarter, what he's saying is that, since the Heliarchs aren't some magic group of Saryds/Kimek/Arachi smarter than those in the Lunarium, they're not inherently better at making any form of tech or equipment. He means that, much like the Heliarchs, they too can get tech of that level, thus closing the gap and giving them a real fighting chance.
Your choice if you want to just remove that portion or change it to something else.

```suggestion
				log "Factions" "Lunarium Redoubt" "The Lunarium is, militarily, at a great disadvantage to the Heliarchs. Their ships are far less durable and their weapons far less powerful. Elirom, however, believes that the Lunarium are smarter than the Heliarchs, and that they are only a few steps away from having the ability to close the technological gaps as well, though Pyakri is skeptical of his words."
```

Reply:
```suggestion
				log "Factions" "Lunarium Redoubt" "The Lunarium is, militarily, at a great disadvantage to the Heliarchs. Their ships are far less durable and their weapons far less powerful. Elirom, however, believes that the Lunarium are only a few steps away from being able to close the technological gaps, though Pyakri is skeptical of his words."
```